pkg/controller/controller_test.go

```diff
@@ -2649,6 +2685,12 @@ func assertServiceBindingOrphanMitigationSuccess(t *testing.T, obj runtime.Objec
 	assertServiceBindingReconciledGeneration(t, obj, originalBinding.Generation)
 	assertServiceBindingInProgressPropertiesNil(t, obj)
 	assertServiceBindingExternalPropertiesNil(t, obj)
+	switch originalBinding.Status.UnbindStatus {
+	case v1beta1.ServiceBindingUnbindStatusNotRequired:
```

Comment:
The original `UnbindStatus` should never be NotRequired. If orphan mitigation suceeded, then orphan mitigation was in progress for the original binding. Hence, an unbind must have been required for the original binding.